paper/paper.md

```diff
@@ -11,16 +11,36 @@ tags:
 authors:
   - name: Ewout ter Hoeven
     orcid: 0009-0002-0805-3425
+    affiliation: "1"
   - name: Jan Kwakkel
     orcid: 0000-0001-9447-2954
+    affiliation: "1"
   - name: Vincent Hess
+    affiliation: "2"
   - name: Thomas D. Pike
     orcid: 0000-0003-1576-0283
+    affiliation: "3"
```

Comment:
Sorry I stink ---could you switch me to George Mason as well

Reply:
No worries, done!